v0/destinations/one_signal/transform.js

```diff
@@ -157,11 +157,11 @@ const trackResponseBuilder = (message, { Config }) => {
         properties[item.propertyName] ||
```

Comment:
if (typeof properties[item.propertyName] === "string") should cover these if (properties[item.propertyName] || properties[item.propertyName] === "") right?

Reply:
yeah, I also think so.

Reply:
Updated.